jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/CommonUtils.java

```diff
@@ -1265,5 +1266,42 @@ public static String eshttpGet(final String url, Map<String, String> headers) th
         }
     }
 
+    public static List<Map<String, String>> extractListOfMaps(String jsonString) throws Exception {

```

Comment:
I wouldn't call it extraction... It's more like converting or reading...
As you did for couple method below

Reply:
Also I don't see any other use for it except in Policy executor class. At least for now. Why don't just move it there and make a private and call exactly what you are using it for? Also in this three methods you creating 2 separate instances of object mapper, while there is one already in PolicyExecutor (I have a comment about it). So it will be less object to create and more to reuse.

Reply:
removed extra comments